rotkehlchen/tests/api/test_nfts.py

```diff
@@ -153,10 +153,10 @@ def test_nft_balances_and_prices(rotkehlchen_api_server):
     response = requests.get(api_url_for(
         rotkehlchen_api_server,
         'nftsbalanceresource',
-    ), json={'async_query': False})
-
+    ), json={'async_query': False, 'ignore_cache': True})
     result = assert_proper_response_with_result(response)
-    for nfts_balances in result.values():
```

Comment:
Isn't this lacking a test with `ignore_cache: False`?

Reply:
Yeah, it does, thx :+1: